backend/resources/gpml/config.edn

```diff
@@ -84,13 +84,45 @@
                                          ]
                                         ["/review" {:middleware [#ig/ref :gpml.auth/auth-middleware
                                                                  #ig/ref :gpml.auth/auth-required
-                                                                 #ig/ref :gpml.auth/approved-user
-                                                                 #ig/ref :gpml.auth/admin-required-middleware]}
+                                                                 #ig/ref :gpml.auth/approved-user]}
                                          ["/get-reviewers"
```

Comment:
as we agree in zulip, move this entrypoint to "/reviewer"

Reply:
Done.